pymc4/inference/utils.py

```diff
@@ -4,8 +4,8 @@
 from .. import Model, flow
 
 
-def initialize_state(
-    model: Model, observed: Optional[dict] = None
+def initialize_sampling_state(
+    model: Model, observed: Optional[dict] = None, state=None
```

Comment:
You forgot to add `state` into the evaluate model call. It should also be noted in the docstring